lang/python/python-package.mk

```diff
@@ -130,7 +130,8 @@ define PyBuild/Compile/Default
 	$(foreach pkg,$(HOST_PYTHON_PACKAGE_BUILD_DEPENDS),
 		$(call host_python_pip_install_host,$(pkg))
 	)
-	$(call Build/Compile/PyMod,, \
+	$(call Build/Compile/PyMod,$(PYTHON_PKG_SETUP_SUBDIR), \
```

Comment:
ack for the `PYTHON_PKG_SETUP_SUBDIR` ; this looks like a good idea
thanks :)

Reply:
This is still there by it's now Build/Setup/PyMod because it's use by both (the now split) build_ext and install